src/couch_views/src/couch_views_jobs.erl

```diff
@@ -59,6 +60,19 @@ build_view_async(TxDb, Mrst) ->
     {ok, JobId}.
 
 
+ensure_correct_tx(#{tx := undefined} = TxDb) ->
+    TxDb;
+
+ensure_correct_tx(#{tx := Tx} = TxDb) ->
+    case erlfdb:is_read_only(Tx) of
+        true -> TxDb#{
```

Comment:
This formatting is awkward. You've multi-lined the map from a single line case statement.

One or the other of these maybe?

```erlang
case erlfdb:is_read_only(Tx) of
    true -> TxDb#{tx := undefined};
    false -> TxDb
end,

case erlfdb:is_read_only(Tx) of
    true ->
        TxDb#{tx := undefined};
    false ->
        TxDb
end
```